eng/pipelines/live-test-cleanup.yml

```diff
@@ -1,7 +1,7 @@
 schedules:
```

Comment:
Can we please move this scheduling into the web ui? The cron job schedules are not easy to maintain across the larger devops org while balancing schedules.